src/azul/indexer/document.py

```diff
@@ -222,23 +227,34 @@ def from_index(self, value: T) -> N:
 
 class PassThrough(Generic[T], FieldType[T, T]):
 
+    def __init__(self, es_type):
+        super().__init__()
+        self.es_type_ = es_type
```

Comment:
Leading underscore for private members, trailing to avoid shadowing. As recommended in pep8.